.bowerrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this necessary? ember-share doesn't use bower anymore

Reply:
poof, gone